src/lib/Rebranding/XmlRebranding.php

```diff
@@ -16,6 +16,33 @@ public function rebrand(string $input): string
 
         $output = $this->replace($this->routeNamesMap, $output);
         $output = $this->replace($this->serviceTagNamesMap, $output);
+        $output = str_replace(array_keys($this->routeNamesMap), array_values($this->routeNamesMap), $output);
```

Comment:
@Nattfarinn I'm not sure if this is just out of scope addition or a typo in the property